core/src/test/scala/integration/kafka/api/BaseConsumerTest.scala

```diff
@@ -36,6 +36,7 @@ import org.apache.kafka.common.errors.WakeupException
  */
 abstract class BaseConsumerTest extends IntegrationTestHarness with Logging {
 
+  val EPSILON = 0.1
```

Comment:
All uppercase is not used in the Scala code, you just capitalise the first letter, but this is just a field in a class, so I'd just be consistent with the other fields like `producerCount`, etc.